backend/src/models/service.ts

```diff
@@ -73,4 +102,68 @@ export class Service extends BaseEntity {
   censysIpv4Results: {
     [x: string]: any;
   };
+
+  /** Wappalyzer output */
+  @Column({
+    type: 'jsonb',
+    default: []
+  })
+  wappalyzerResults: {
+    name: string;
+    slug: string;
+    version: string;
+    icon: string;
+    website: string;
+    confidence: number;
+    cpe?: string;
+    categories: {
+      name: string;
+      slug: string;
+      id: number;
+    }[];
+  }[];
+
+  @BeforeInsert()
+  @BeforeUpdate()
+  setProducts() {
+    const products: { [cpe: string]: Product } = {};
+    const misc: Product[] = [];
```

Comment:
What do you think `misc` should be used for?

Reply:
I forgot to add it to the array at the end, fixed now